features/admin.remote-userstores.v1/components/edit/userstore-general-settings.tsx

```diff
@@ -517,354 +516,356 @@ export const UserStoreGeneralSettings: FunctionComponent<UserStoreGeneralSetting
-                                                    className={
-                                                        isMobileViewport
-                                                            ? "mb-1x mt-1x inline-button button-width"
-                                                            : "inline-button"
-                                                    }
-                                                >
-                                                    <PrimaryButton
-                                                        type="submit"
-                                                        disabled={ !(userStore?.properties?.find(
-                                                            (property: UserStoreProperty) =>
-                                                                property.name === DISABLED)?.value === "false") }>
-                                                        { t("common:update") }
-                                                    </PrimaryButton>
-                                                </div>
+                                            disabled={ isReadOnly || isDisabled }
+                                        >
+                                                Regenerate token
+                                        </Button>
+                                    </List.Content>
+                                    <List.Content>
+                                        <List.Header>
+                                            { resolveConnectionStatusIcon(connection?.connected) }
+                                            <strong>{ connection?.agent?.displayName }</strong>
+                                        </List.Header>
+                                        <List.Description className="mt-2 ml-1">
+                                            { AGENT_CONNECTION_DESCRIPTION }
+                                        </List.Description>
+                                    </List.Content>
+                                </List.Item>
+                            )) }
+                            { agentConnections.length === 1 && (
+                                <List.Item columns={ 2 } verticalAlign="middle">
+                                    <List.Content floated="right">
+                                        <Button
+                                            color="orange"
+                                            onClick={ handleGenerateToken }
+                                            disabled={ isReadOnly || isDisabled }
+                                        >
+                                                Generate token
+                                        </Button>
+                                    </List.Content>
+                                    <List.Content>
+                                        <List.Header>
+                                            <Icon name="times circle" color="red" className="mr-1" />
```

Comment:
Let's avoid the utility classes like `mr-1`. Check the other places as well.

Reply:
These are already available. The component is separated out.
https://github.com/wso2/identity-apps/blob/26842aef8a1ce26fbad135669539133e2299ca9c/features/admin.remote-userstores.v1/components/edit/userstore-general-settings.tsx#L208